src/main/java/minicraft/core/Game.java

```diff
@@ -105,24 +105,26 @@ public static void main(String[] args) {
 				}
 			}
 		});
-		
+
 		Analytics.GameStartup.ping();
 
 		Initializer.parseArgs(args); // Parses the command line arguments
-		
+
 		input = new InputHandler(Renderer.canvas);
-		
+
 		Tiles.initTileList();
 		Sound.init();
 
 		World.resetGame(); // "half"-starts a new game, to set up initial variables
 		player.eid = 0;
 		new Load(true); // This loads any saved preferences.
+		MAX_FPS = (int) Settings.get("fps");
 
-    Localization.loadLanguage();
+		Localization.changeLanguage((String)Settings.get("language"));
```

Comment:
Why did you add this? Please remove it.

Reply:
I think this resolved that these settings are not being applied in setup.

Reply:
No, I have already fixed that. Take a look at the line below.

Reply:
But I haven't seen that Localization.loadLocalizatiom get from settings.

Reply:
Okay, good

Reply:
Done.